trace_context/TRACE_STATE_REGISTRY.md

```diff
@@ -0,0 +1,7 @@
+# Trace State Registry
+
+| Trace State Name | Verdor / Project Name | website |
+| - | :-: | - |
+| sw | Apache SkyWalking (Incubating) | http://skywalking.io/ |
```

Comment:
fyi if we had such a registry, the table should point to the actual format.

basically as a developer or even a user, if I go to a table for a trace state entry, it should bring me exactly to the state definition... remember this? https://drive.google.com/drive/u/0/folders/0B0tSnQT3uGdAWU00QW1HQUFGeG8 (not sure you can see it)

![screen shot 2018-03-19 at 3 41 13 pm](https://user-images.githubusercontent.com/64215/37583347-0341ab00-2b8c-11e8-907e-b8d36a88ef91.png)